skillmap/src/lib/readyResources.ts

```diff
@@ -16,10 +16,12 @@ export class ReadyPromise {
     private promise_: Promise<ReadyResources>
     private resources: ReadyResources;
     private appMounted?: boolean;
+    public cloudSyncCheckHasFinished: boolean
```

Comment:
Odd to have `cloudSyncCheckHasFinished` in here. It isn't part of the ready check. It's only used by App.tsx. I think it could live on AppImpl.